builtin/providers/kubernetes/resource_kubernetes_config_map.go

```diff
@@ -21,6 +21,8 @@ func resourceKubernetesConfigMap() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			State: schema.ImportStatePassthrough,
 		},
+		Name:        "Config Map",
```

Comment:
I don't think it will always be that easy. Sometimes we simplify the machine-friendly name or it's tricky to machine-convert it because of casing, e.g. `aws_config_configuration_recorder` should be called `AWSConfig Configuration Recorder`, `sfn_activity` should IMO be called `Step Function Activity` etc.